hs-bindgen/src-internal/HsBindgen/C/Environment.hs

```diff
@@ -0,0 +1,75 @@
+-- | Handle `libclang`-specific environment variables.
+
+module HsBindgen.C.Environment (
```

Comment:
`HsBindgen.C` is a wrong namespace.

This is not C-*AST* related.

Reply:
Can you suggest a better namespace?

Reply:
`HsBindgen.Environment`? Would that work?

Reply:
This is clang specific, so i'd expect seeing `Clang` (or rather `ClangArgs`) in the module name.

Reply:
.. so maybe something like `HsBindgen.Clang.Args` as an extension of `Clang.Args`.

But to be honest, the only `hs-bindgen` specific bit is the environment variable name (and maybe tracer type in the future). Both specificities could be abstracted away.

Reply:
Ok, thanks for the suggestion. I will move it to `HsBindgen.Clang.Args`. The only dawback I see is that potentially other supported env vars in the future do not have a space there. But we can extend and move the module when the need arises.